src/test/interface/laplace_test.cpp

```diff
@@ -137,3 +184,21 @@ TEST_F(CmdStan, laplace_jacobian_adjust) {
     ASSERT_EQ(sample1.coeff(n, 2), sample2.coeff(n, 2));
   }
 }
+
+TEST_F(CmdStan, laplace_unconstrain) {
+  std::stringstream ss;
+  ss << convert_model_path(simplex_model) << " random seed=1234"
+     << " method=laplace mode=" << convert_model_path(simplex_mode_csv)
+     << " 2>&1";
+  std::string cmd = ss.str();
+  run_command_output out = run_command(cmd);
+  ASSERT_FALSE(out.hasError);
+
+  ss.str(std::string());
+  ss << convert_model_path(simplex_model) << " random seed=1234"
+     << " method=laplace mode=" << convert_model_path(simplex_mode_bad_csv)
+     << " 2>&1";
+  cmd = ss.str();
+  out = run_command(cmd);
+  ASSERT_TRUE(out.hasError);
```

Comment:
Question: does `out.hasError` distinguish between a "nice" error (one we throw) and something like a segmentation fault?

Reply:
no, and it should.   fixing.

Reply:
no it doesn't.   adding some logic to check for that.